httpcore5/src/main/java/org/apache/hc/core5/reactor/InternalDataChannel.java

```diff
@@ -243,7 +243,8 @@ public void execute(final SSLIOSession sslSession) {
                         }
                     }
 
-                }))) {
+                },
+                null))) {
```

Comment:
@rschmitt Do we really need an extra constructor parameter if it is always set to `null`? Otherwise looks good.

Reply:
The `null` argument here is temporary. Wiring this value up all the way from `InternalHttpAsyncExecRuntime` is going to require tons of additional changes.